### PR TITLE
 chore: more rename to horaedb

### PR DESCRIPTION
**Pull request by chunshao90**

### CODE REVIEW

File: docs/src/en/dev/profiling.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ Example:
...
 
-// run ceresdb-server
-./ceresdb-server ....
+// run horaedb-server
+./horaedb-server ....
 
 // 60s cpu sampling data
 curl -L '0:5000/debug/profile/heap/60' > /tmp/heap_profile
-jeprof --show_bytes --pdf /usr/bin/ceresdb-server /tmp/heap_profile > profile_heap.pdf
+jeprof --show_bytes --pdf /usr/bin/horaedb-server /tmp/heap_profile > profile_heap.pdf
```
Comment (ShiKaiWi) [inline docs/src/en/dev/profiling.md:38]: Is the binary name changed?

File: docs/src/en/operation/system_table.md [hunk 2]
```diff
@@ -32,8 +31,8 @@ curl --location --request POST 'http://localhost:5000/sql' \
     "rows":[
         {
             "timestamp":0,
-            "catalog":"ceresdb",
-            "schema":"monitor_trace",
+            "catalog":"horaedb",
```
Comment (ShiKaiWi) [inline docs/src/en/operation/system_table.md:34]: Is the default catalog name changed?

File: docs/src/en/operation/system_table.md [hunk 3]
```diff
@@ -32,8 +31,8 @@ curl --location --request POST 'http://localhost:5000/sql' \
...
+            "schema":"public",
```
Comment (ShiKaiWi) [inline docs/src/en/operation/system_table.md:35]: Why is the old schema `monitor_trace`? If it is called `monitor_trace`, should it be changed?

Comment (chunshao90) [reply docs/src/en/operation/system_table.md:35]: I think the previous demo was wrong.

File: docs/src/en/operation/system_table.md (hunk 2 above)
Comment (chunshao90) [reply docs/src/en/operation/system_table.md:34]: Yes.

**Review by tanruixiang — APPROVED** [review]
LGTM

**Review by tisonkun — APPROVED** [review]

**Review by waynexia — APPROVED** [review]

**Pull request merged**